src/Databases/MySQL/MaterializeMySQLSyncThread.h

```diff
@@ -63,6 +64,10 @@ class MaterializeMySQLSyncThread
     MaterializeMySQLSettings * settings;
     String query_prefix;
 
+    const int ER_ACCESS_DENIED_ERROR = 1045;
```

Comment:
It's unclear where are these error codes from. Please add comment or include `mysqld_error.h`

Reply:
Ok , I will add comment. `include <mysqld_error.h>` maybe not so suitable.